docs/content/latest/benchmark/tpcc-ysql/_index.md

```diff
@@ -31,6 +31,15 @@ isTocNested: true
 ## Overview
 Follow the steps below to run the open-source [oltpbench](https://github.com/oltpbenchmark/oltpbench) TPC-C workload against YugabyteDB YSQL. [TPC-C](http://www.tpc.org/tpcc/) is a popular online transaction processing benchmark that provides metrics you can use to evaluate the performance of YugabyteDB for concurrent transactions of different types and complexity that are either either executed online or queued for deferred execution.
 
+### Results at a glance
+| Warehouses| TPMC | Efficiency
```

Comment:
* Can we rename `Efficiency` to `Efficiency (approx)`?

* We should also add the following column:

**Cluster Size**
The values for this should look like the following:
3 nodes of type `c5d.4xlarge` (16 vCPUs)

* Also, can we change the 10 warehouses example to a 2 core machine? Say a c5.large or something like that. Since its just an example (assuming our efficiency will be close to max here).

Reply:
Done.